docs/json.md

```diff
@@ -135,7 +135,7 @@ The annotation
 ```abap
 "! $multipleOf value
 ```
-ensures that values of a component described by this ABAP Doc comment can only be a multiple of the provided value.
+ensures that values of a component described by this ABAP Doc comment can only be a multiple of the provided value. Note that it can only be used for integers.
```

Comment:
Make diff more comprehensive.
```suggestion
ensures that values of a component described by this ABAP Doc comment can only be a multiple of the provided value.
Note that it can only be used for integers.
```